index.js

```diff
@@ -2,17 +2,17 @@
 /*!
  * Adds dynamically-updated docs as /explorer
  */
-var url = require('url');
-var path = require('path');
-var urlJoin = require('./lib/url-join');
-var _defaults = require('lodash').defaults;
-var cors = require('cors');
-var createSwaggerObject = require('loopback-swagger').generateSwaggerSpec;
-var SWAGGER_UI_ROOT = require('strong-swagger-ui/index').dist;
-var STATIC_ROOT = path.join(__dirname, 'public');
-
-module.exports = explorer;
-explorer.routes = routes;
+  var url = require('url');
+  var path = require('path');
+  var urlJoin = require('./lib/url-join');
+  var _defaults = require('lodash').defaults;
+  var cors = require('cors');
+  var createSwaggerObject = require('loopback-swagger').generateSwaggerSpec;
+  var SWAGGER_UI_ROOT = require('strong-swagger-ui/index').dist;
+  var STATIC_ROOT = path.join(__dirname, 'public');
+
+  module.exports = explorer;
+  explorer.routes = routes;
```

Comment:
@Amir-61 @superkhau this has re-indented whole `index.js` by one level of nesting :(